docs/get-started/features.md

```diff
@@ -291,6 +304,26 @@ Matching is done by performing logical _AND_ for each provided Element, i.e the
 Rule will match if all provided Elements (kernel config options) are enabled
 (`y` or `m`) or matching `=<value>` in the kernel.
 
+##### Nodename Rule
+
+###### Nomenclature
+
+```
+Element     :A nodename regexp pattern
+```
+
+The Rule allows matching the node's name against a provided list of Elements.
```

Comment:
Should a note be added here clarifying that this matches on the node's name as registered in the kubernetes cluster ?
or do you think its clear they way it is now ?

Reply:
IMHO node name is clear...?

Reply:
ack